velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -129,5 +128,43 @@ TEST_F(StringAggregatesTest, nonFixedWidthAggregate) {
   testWindowFunction(input, "max(c2)", kOverClauses);
 }
 
+class CountAggregateTest : public WindowTestBase {};
+
+TEST_F(CountAggregateTest, basic) {
+  testWindowFunction({makeSimpleVector(10)}, "count(c2)", kOverClauses);
+}
+
+TEST_F(CountAggregateTest, singlePartition) {
+  testWindowFunction(
+      {makeSinglePartitionVector(50), makeSinglePartitionVector(40)},
+      "count(c2)",
+      kOverClauses);
+}
+
+TEST_F(CountAggregateTest, singleRowPartitions) {
+  testWindowFunction(
+      {makeSingleRowPartitionsVector(40)}, "count(c2)", kOverClauses);
+}
+
+TEST_F(CountAggregateTest, emptyFrames) {
+  auto c0 = makeFlatVector<int64_t>({-1, -1, -1, -1, -1, -1, 2, 2, 2, 2});
+  auto c1 = makeFlatVector<double>({-1, -2, -3, -4, -5, -6, -7, -8, -9, -10});
+  auto input = makeRowVector({c0, c1});
+
+  auto expected = makeRowVector(
```

Comment:
I feel we should change the tests in another way: 

- AggregateWindowTestBase works with kOverClauses and kFrameClauses. So remove testWindowFunction(input, kEmptyFrameClauses); in singlePartition. Then AggregateWindowTestBase works with all aggregate functions including count.
- Add a AggregateEmptyFramesTest with 2 functions:
i) Test that iterates over kEmptyFrameClauses and kOverClauses for all non-count functions
ii) Function countEmptyFrames which is the code you added in function emptyFrames here.